charts/images.yaml

```diff
@@ -49,7 +49,7 @@ images:
 - name: csi-provisioner
   sourceRepository: github.com/kubernetes-csi/external-provisioner
   repository: k8s.gcr.io/sig-storage/csi-provisioner
-  tag: "v2.1.2"
+  tag: "v3.0.0"
```

Comment:
From the release [notes](https://github.com/kubernetes-csi/external-provisioner/blob/release-3.0/CHANGELOG/CHANGELOG-3.0.md#no-really-you-must-read-this-before-you-upgrade)
>    Update external-snapshotter dependency to v4, enabling use of the volume snapshot v1 client, which requires minimum Kubernetes version 1.20 (#650, @xinyanw409)

So k8s clusters running on 1.18 and 1.19 cannot get this update.

Reply:
Hi @vpnachev, I added a targetVersions to support the differences before and after k8s v1.20

Please let me know if anything else should be done.